src/Ex4.java

```diff
@@ -0,0 +1,26 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class Ex4 {
+
+        public static void main(String[] args) {
+
+            String[] names = { "Elena", "Thomas", "Hamilton", "Suzie", "Phil", "Matt", "Alex","Emma", "John", "James", "Jane", "Emily", "Daniel", "Neda", "Aaron", "Kate" };
+            int[] time = { 341, 273, 278, 329, 445, 402, 388, 275, 243, 334, 412, 393, 299, 343, 317, 265 };
+            int n = time.length;
```

Comment:
n - not a good name to the variable. timeArrayLength - good name. Moreover, you have used n only once, and this occurrence can be replaced with time.length